src/ip_address.h

```diff
@@ -79,6 +81,13 @@ class IPAddress {
                 m_family = AF_UNSPEC;
             }
             break;
+        case AF_UNIX:
+            if (len >= sizeof(sockaddr_un)) {
+                m_addr_un = std::string(reinterpret_cast<const sockaddr_un *>(sa)->sun_path);
```

Comment:
```suggestion
                m_addr_un = unix_sockaddr_to_string(reinterpret_cast<const sockaddr_un *>(sa));
```

Reply:
I agree but again we have the problem with `common.h` -> `defs.h` -> `ip_address.h`

so all the changes for feedfile I can use common.h 
but I can't use `unix_sockaddr_to_string` func in common.h

Reply:
Try moving constructor to cpp file and include "common.h" inside cpp.